framework/contracts/native/account-factory/src/commands.rs

```diff
@@ -32,6 +33,7 @@ pub const CREATE_ACCOUNT_MANAGER_MSG_ID: u64 = 1u64;
 pub const CREATE_ACCOUNT_PROXY_MSG_ID: u64 = 2u64;
 
 /// Function that starts the creation of the Account
+#[allow(clippy::too_many_arguments)]
```

Comment:
We have 6 arguments of create_account action, I guess we can pack them into struct, wdyt?